vendor/github.com/grafana/dskit/ring/lifecycler.go

```diff
@@ -43,23 +44,23 @@ type LifecyclerConfig struct {
 	ReadinessCheckRingHealth bool          `yaml:"readiness_check_ring_health" category:"advanced"`
 
 	// For testing, you can override the address and ID of this ingester
-	Addr string `yaml:"address" doc:"hidden"`
-	Port int    `doc:"hidden"`
-	ID   string `doc:"hidden"`
+	Addr string `yaml:"address" category:"advanced"`
```

Comment:
do you know how one can use this new category feature?

Reply:
I asked Christian who worked with the Mimir team. These tags are used by the doc-generator and the help messages to categorise settings. Loki/GEL have a separate doc-generator so the tags are ignored at the moment.